src/lib/util/wizard-step.interface.ts

```diff
@@ -65,6 +71,11 @@ export abstract class WizardStep {
    */
   public optional = false;
 
+  /**
+   * A boolean describing if the wizard step is an disabled step
+   */
+  public disabled = false;
```

Comment:
I don't understand what a disabling a step does?

Reply:
Preventing you from entering this step

Use case:
- you need to choose between 2 options each option make different step enabled

I needed to use this option actually

Reply:
Not precisely, because disabled is a state, it’s need to have a style and custom behavior:

Like when going to next step it will jump over the disabled one.

And other libraries have this too...

Reply:
Ok, I understand.
Then the questions are:
- how do you define a step as disabled?
- how should the wizard react when the user wants to navigate to a disabled step?
   Is it always possible to skip the step? If yes, in which direction? Are there other actions the wizard could do in instead of entering a disabled step?

In any case I think supporting disabled steps should be moved to its own PR.